src/main/java/com/thomashan/coup/turn/state/TurnState.java

```diff
@@ -49,9 +49,9 @@
     // FIXME: replace with getAllowableActions
     List<ActionType> getAllowableActionTypes();
 
-    ActionState performAction(A Action);
+    TurnState performAction(A Action);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method parameter name 'Action' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4558920)